source/api/cypress-api/selector-playground-api.md

```diff
@@ -19,7 +19,7 @@ Cypress.SelectorPlayground.getSelector($el)
 
 **{% fa fa-angle-right %} options**  ***(Object)***
 
-An object containing one or both of the following:
+An object containing any or all of the following options:
```

Comment:
In testing I discovered that you can pass an _empty_ object as an argument. I dunno why one would want to do that but you can so I tweaked the docs to be a little more permissive.